gprofiler/containers_client.py

```diff
@@ -49,6 +49,7 @@ def get_container_name(self, pid: int) -> str:
             return ""
 
         if pid in self._pid_to_container_name_cache:
+            self._current_container_names.add(self._pid_to_container_name_cache[pid])
             return self._pid_to_container_name_cache[pid]
```

Comment:
```suggestion
            name = self._pid_to_container_name_cache[pid]
            self._current_container_names.add(name)
            return name
```

nicer w/o accessing it twice